modules

```diff
@@ -1,5 +1,5 @@
 GLUON_SITE_FEEDS='ffmuc'
 
-PACKAGES_FFMUC_REPO=https://github.com/krombel/gluon-packages.git
+PACKAGES_FFMUC_REPO=https://github.com/freifunkMUC/gluon-packages.git
 PACKAGES_FFMUC_BRANCH=add_domain-migration-ffmuc
```

Comment:
I guess PACKAGES_FFMUC_BRANCH should be "master"?

Reply:
Ohh yes of course. Thanks for pointing it out!